tests/network/flat_overlay/test_flat_overlay.py

```diff
@@ -20,6 +20,7 @@ class TestFlatOverlayConnectivity:
     @pytest.mark.gating
     @pytest.mark.ipv4
     @pytest.mark.polarion("CNV-10158")
+    # Not marked as `conformance`; requires NMState
```

Comment:
If I'm not wrong flat overlay doesn't require manipulations on node interfaces, so we don't use nmstate here. I also followed the flow of this test and indeed didn't find using nmstate down the path.
Did you try running it on a cluster without nmstate and it failed because of that?